src/Grpc/JsonTranscoding/src/Shared/ServiceDescriptorHelpers.cs

```diff
@@ -451,6 +451,30 @@ public static bool TryResolvePattern(HttpRule http, [NotNullWhen(true)] out stri
         return null;
     }
 
+    public static FieldDescriptor? ResolveResponseBodyDescriptor(string responseBody, MethodDescriptor methodDescriptor)
+    {
+        if (!string.IsNullOrEmpty(responseBody))
+        {
+            FieldDescriptor? responseBodyDescriptor = null;
+            if (!string.IsNullOrEmpty(responseBody))
```

Comment:
Any way we can restructure the code so that we're not checking `string.IsNullOrEmpty(responseBody)` twice? It seems like we'll always return null if the check fails.